src/StructuralAnalyses/LinearStaticAnalyses.jl

```diff
@@ -111,7 +119,7 @@ function step!(sa::LinearStaticAnalysis)
     cg!(ΔU, K, fₑₓₜ_red)
 
     # Update displacements into the state.
-    state.Uᵏ[free_dofs_idx] .+= ΔU
+    displacements(state)[free_dofs_idx] .= ΔU
```

Comment:
How is `displacements(state)[free_dofs_idx]` different from `Δ_displacements(state)`? At the `free_dofs_idx`?

Reply:
Is the same, the thing is that in linear analysis you solve K / F = U and thats it. So we dont have to sum at each time step. Moreover, I used the getter function instead of .field